ide/dune-project

```diff
@@ -0,0 +1,4 @@
+(lang dune 1.0)
+
+(name coqide-devel)
+(version 8.9+alpha)
```

Comment:
Hopefully this will get automatic from the tags once https://github.com/ocaml/dune/issues/880 lands upstream, so no need to update now.

Reply:
Well I disagree strongly on this one because you cannot guarantee when this will happen so in the meantime it is important to document (it's generally a fallacious argument to say "I don't document because it will be different in the future").

Reply:
The easier fix here is just to remove the version field then.